content/tracing/advanced_usage/custom_tagging.md

```diff
@@ -44,6 +44,46 @@ class ServletImpl extends AbstractHttpServlet {
 
 {{% /tab %}}
 {{% tab "Python" %}}
+
+**Adding tags to a span**
+
+You can add tags directly to a span by calling `set_tag`. For example with the following Flask route handler:
+
+```python
+from ddtrace import tracer
+
+@app.route('/post/<int:post_id>')
+def handle_post(post_id):
+  with tracer.trace('web.request') as span:
+    span.set_tag('post.id', post_id)
+```
+
+**Adding tags to a current active span**
+
+The current span can be retrieved from the context in order to set tags. This way, if a span was started by our instrumentation, you can retrieve the span and add custom tags. Note that if a span does not exist, `None` will be returned:
+
+```python
+from ddtrace import tracer
+
+@app.route('/post/<int:post_id>')
+@tracer.wrap()
+def handle_post(post_id):
+  # get the active span in the context
+  current_span = tracer.get_span()
```

Comment:
It's `tracer.current_span` property: https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/tracer.py#L288